src/lando/lando/settings.py

```diff
@@ -10,24 +10,41 @@
 https://docs.djangoproject.com/en/dev/ref/settings/
 """
 
+
+import os
 from pathlib import Path
 
-# Build paths inside the project like this: BASE_DIR / 'subdir'.
-BASE_DIR = Path(__file__).resolve().parent.parent
 
+def get_setting(
+    key: str, default: any, source: any = os.environ, target_type=None
+) -> any:
```

Comment:
We need to use `typing.Any` here, using the `any` builtin function isn't supported.